src/components/CameraControls/index.tsx

```diff
@@ -31,9 +41,11 @@ const CameraControls = ({
     zoomOut,
     setPanningMode,
     setFocusMode,
+    setCameraType,
 }: CameraControlsProps): JSX.Element => {
     const [isFocused, saveFocusMode] = useState(true);
     const [mode, setMode] = useState(ROTATE);
+    const [camera, setCamera] = useState(PERSPECTIVE);
```

Comment:
a more descriptive name might be `cameraProjectionType`

Reply:
...I see `setCameraType` was already used elsewhere, which is still better than `setCamera`.